extension/googleclientauthextension/metadata.yaml

```diff
@@ -9,3 +9,8 @@ status:
 
 tests:
   skip_lifecycle: true
+  goleak:
```

Comment:
I'm not familiar with the use of `metadata.yaml` file in this repo, so just confirming - is this required when the `generated_package_test.go` already programmatically seem to ignore the goleak coming from `opencensus` ?

Reply:
`generated_package_test.go` is generated based on `metadata.yaml`. It is required to write `goleak.ignore` block here.

Without this `goleak.ignore`, the tests should fail with goroutine leak caused by `opencensus`.

Reply:
I see, thanks for the confirmation.